src/main/java/net/dv8tion/jda/api/JDA.java

```diff
@@ -2374,6 +2375,27 @@ default List<Emote> getEmotesByName(@Nonnull String name, boolean ignoreCase)
      * @see #shutdown()
      */
     void shutdownNow();
+    
+    /**
+     * Blocks until this JDA instance fully shuts down or the timeout expires. <b>This method does not shut down JDA.</b>
+     * 
+     * <p>If you want to run code after shutting down async, use the {@link net.dv8tion.jda.api.events.ShutdownEvent ShutdownEvent}
+     * in preference to this method.
+     * 
+     * @param  timeout
+     *          The timeout to wait for shutdown before returning.
+     *        
+     * @param  unit
+     *          {@link TimeUnit Unit} for the timeout value.
+     *        
+     * @throws InterruptedException
+     *          if the current thread is interrupted while waiting.
```

Comment:
```suggestion
     * @param  timeout
     *         The timeout to wait for shutdown before returning.
     *        
     * @param  unit
     *         {@link TimeUnit Unit} for the timeout value.
     *        
     * @throws InterruptedException
     *         if the current thread is interrupted while waiting.
```

I had 1 space too much on the throws description, and you copied that over + added a space to the other descriptions which is incorrect